ci-operator/step-registry/platform-external/cluster/wait-for/ccm-nodes-initialized/platform-external-cluster-wait-for-ccm-nodes-initialized-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: platform-external-cluster-wait-for-ccm-nodes-initialized
+  from: cli
+  commands: platform-external-cluster-wait-for-ccm-nodes-initialized-commands.sh
+  grace_period: 10m
+  timeout: 35m
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
```

Comment:
it's missing the env spec, I am going to commit after the running rehearsal finishes.